pilot/pkg/config/kube/crd/controller/controller.go

```diff
@@ -99,7 +118,9 @@ func NewController(client *Client, options controller2.Options) model.ConfigStor
 
 	// add stores for CRD kinds
 	for _, s := range client.Schemas().All() {
-		out.addInformer(s, options.WatchedNamespace, options.ResyncPeriod)
+		if crdExistsWithRetry(client, s) {
```

Comment:
Could u add a warning log in else branch here?

Reply:
Yes, done. Thanks